index.js

```diff
@@ -1,21 +1,101 @@
 // TODO: import files from includes
-// const file_one = require("./includes/file_one");
-// const file_two = require("./includes/file_two");
+const fivetranLogAccount = require("./includes/account");
+const fivetranLogAccountMembership = require("./includes/account_membership");
+const fivetranLogActiveVolume = require("./includes/active_volume");
+const fivetranLogConnector = require("./includes/connector");
+const fivetranLogCreditsUsed = require("./includes/credits_used");
+const fivetranLogDestination = require("./includes/destination");
+const fivetranLogDestinationMembership = require("./includes/destination_membership");
+const fivetranLogLog = require("./includes/log");
+const fivetranLogUser = require("./includes/user");
+const fivetranLogConnectorDailyApiCalls = require("./includes/connector_daily_api_calls");
+const fivetranLogConnectorStatus = require("./includes/connector_status");
+const fivetranLogCreditMarHistyory = require("./includes/credit_mar_history");
+const fivetranLogMarTableHistory = require("./includes/mar_table_history");
 
 module.exports = (params) => {
 
-    params = {
-      // TODO: set default params
+  params = {
+    // set defaults for parameters
+    fivetranLogFirstDate: "2016-01-01",
+    fivetranLogSchema: "fivetran_log",
 
-      ...params
-    };
+    ...params
+  };
 
-    // Publish and return datasets.
-    let result = {
+  let account, account_membership, active_volume, connector, credits_used, destination, destination_membership, log, user;
+
+  account = declare({
+    ...params.defaultConfig,
+    schema: params.fivetranLogSchema,
```

Comment:
I would consider adding support for overriding the `fivetranLogDatabase` too, as I suspect it's not uncommon to have these logs go into a different project/database than their default dataform one!

Reply:
I'm going to save that for a future update, but I do agree